pkg/logging/logging.go

```diff
@@ -20,6 +20,16 @@ var (
 	repeatLogs bool   = false
 )
 
+// GetLogWriter gives access to `logWriter` without the possibility of changing it
+func GetLogWriter() io.PipeWriter {
```

Comment:
return type should be generic `io.Writer`, of this package don't have to know that it's a pipe
```suggestion
func GetLogWriter() io.Writer {
```

Reply:
fixed